content/birds/patricia.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Patricia Realini
+sub: Technical Project Manager
+bird: patricia
+date: 2020-11-02
+image:
```

Comment:
@patriciarealini you can check the [deploy preview](https://5fb6f77ee3041900071ca631--oddleventy.netlify.app/authors/patricia/) or run this branch locally and go to `localhost:8080/authors/patricia/` to see how the image is sized/aligned. It looks like it needs a bit of adjustment. A few notes on that:

- Size: the `height` and `width` properties here should match the actual pixel size of the source image. That's used to give it an intrinsic ratio, and also to generate appropriate smaller versions for `srcset`.
- Alignment: you can crop the source as desired, or use a `position` property to set the CSS `object-position`, which defaults to `center`. (it looks to me like you might want to crop some, and then also adjust the position)

Reply:
thanks for the details and breakdown, i ended up following the suggestions in your second bullet point. let me know if there are any further clean ups i need to do.

Reply:
looks great to me. 👍